packages/sanity/src/core/store/key-value/backends/resolve.ts

```diff
@@ -4,5 +4,6 @@ import {memoryBackend} from './memory'
 import {type Backend} from './types'
 
 export function resolveBackend(): Backend {
+  //TODO: add check for "server"
```

Comment:
nit: is this still relevant?

Reply:
Thank you, good catch! Made a note of it for us internally at SDX-1188.